### PR TITLE
Nfs 282 update the dependencies

### PR DESCRIPTION
**Pull request by FenziRocks**
This PR is to update all dependencies, to keep the plugin secure.

**to test**

- Have look on the package.json
- run the tests
- run yarn audit

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,20 +25,23 @@
   "author": "Hassaan Zaidi <hassaan@staffbase.com>",
   "license": "ISC",
   "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.4.3",
+    "@babel/preset-env": "^7.0.0",
     "@staffbase/eslint-config-staffbase": "^0.1.1",
-    "babel-cli": "^6.24.1",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (Ninerian) [inline package.json]: 7.4.3 is the latest version and also installed via the ^ notation. So set it to this version here.

Comment (FenziRocks) [reply package.json]: It happend automatically by npx upgrade bable...

File: package.json [hunk 2]
```diff
@@ -25,20 +25,23 @@
...
+    "babel-jest": "^24.7.1",
     "dmd-bitbucket": "^0.1.10",
     "eslint": "^5.16.0",
     "eslint-config-google": "^0.12.0",
-    "handlebars": "^4.0.6",
+    "handlebars": "^4.1.1",
     "jest": "^24.7.1",
     "jsdoc-to-markdown": "^4.0.1",
-    "node-rsa": "^1.0.5",
+    "node-rsa": "^0.4.2",
```
[the remark this answers is not in the document]
Comment (FenziRocks) [reply package.json]: Merge failure. Probably.

File: package.json (hunk 1 above)
Comment (FenziRocks) [reply package.json]: image

**Pull request merged**